app/assets/javascripts/service_banner.js

```diff
@@ -0,0 +1,20 @@
+document.addEventListener("turbolinks:load", function() {
+    let buttons = document.getElementsByClassName('close-service-notification');
+
+    for (let item of buttons) {
+        if(localStorage.getItem(item.id)) {
```

Comment:
am I missing something? localStorage? Do all our browsers support it?

Reply:
We want the data to remain longer than just a 'session'